CompetitionTemplate/src/competition/subsystems/arm/arm_commands/LowerArmCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.arm.arm_commands;
+import competition.subsystems.arm.ArmSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class LowerArmCommand extends BaseCommand {
+    ArmSubsystem armSubsystem;
+
+    public LowerArmCommand(ArmSubsystem armSubsystem) {
+        this.armSubsystem = armSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        armSubsystem.armMotorPower(-1);
```

Comment:
we should have DoubleProperties for these powers like we did for the collector (lower and raise powers)